etc/firefox.profile

```diff
@@ -28,5 +28,11 @@ include whitelist-usr-share-common.inc
 # private-etc must first be enabled in firefox-common.profile
 #private-etc firefox
 
+dbus-user filter
+dbus-user.own org.mozilla.firefox.*
```

Comment:
Starting with Firefox 75, there should probably also be a `dbus-user.own org.mpris.MediaPlayer2.firefox.*` for hardware media key support.

Reply:
I wanted to start minimal, there a lot more path for firefox + extensions. The policy used by the inoffical firefox-flatpak:
```
dbus-user filter
dbus-user.own org.mozilla.Firefox
dbus-user.own org.mozilla.firefox.*
dbus-user.talk org.freedesktop.FileManager1
dbus-user.talk org.freedesktop.Notifications
dbus-user.talk org.freedesktop.ScreenSaver
dbus-user.talk org.gnome.SessionManager
dbus-system filter
dbus-system.talk org.freedesktop.NetworkManager
```

Reply:
Whoah, `org.freedesktop.NetworkManager` sounds quite evil in this context (MITM via adding a VPN, or leaking IP and MAC addresses of interfaces otherwise isolated with `--net=`.